drivers/hmis/app/models/hmis/hud/custom_assessment.rb

```diff
@@ -76,6 +76,10 @@ class Hmis::Hud::CustomAssessment < Hmis::Hud::Base
     joins(:project).merge(Hmis::Hud::Project.where(id: project_ids))
   end
 
+  scope :with_form_definition_identifier, ->(form_identifiers) do
```

Comment:
One risk I haven't wrapped my head all the way around yet: the form definition identifier is not unique, because forms can have multiple versions. I think this okay, right? We would just return all assessments whose definition identifier matches, regardless of version, which is the desired behavior from the frontend perspective?

I'd like to add spec tests for the changes I'm making before this PR gets merged, but I would like to confirm the general approach first.

Reply:
Sorry I missed this comment the first time around somehow! 

Yes, that is correct. If a client has 2 assessments, one that was collected with SPDAT V1 and another that was collected with SPDAT V2, they should both be returned when they select Filter option "SPDAT" for assessment type.

Right now this means looking at the form_processor.definition_id, but if/when https://github.com/open-path/Green-River/issues/5709 is implemented (which I [propose](https://docs.google.com/document/d/1iL6yr_2clGEW1scCrnO0mt9uZmb_CbP7q7D0c5pqeUE/edit#heading=h.gsie1zz6or3n) we do..) then this query would be simplified to just look at the custom_assessment.definition_identifier.

Reply:
thank you!